contrib/!lang/ruby/packages.el

```diff
@@ -32,6 +32,14 @@
     :config (add-hook 'enh-ruby-mode-hook
                       (lambda () (rbenv-use-corresponding)))))
 
+(defun ruby/init-chruby ()
+  "Initialize chruby mode"
+  (use-package chruby
+    :defer t
+    :init (chruby)
```

Comment:
I originally didn't have this line and it was not working. I am new to emacs and elisp so am probably missing something in my mental model, but it didn't work without this line...